apis/refs/v1beta1/apigeeorganizationref.go

```diff
@@ -12,14 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package v1alpha1
+package v1beta1
```

Comment:
Ah I think this bit me in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3605